public/emojis.js

```diff
@@ -434,43 +533,56 @@ const emojis = {
   TRAM: "🚡",
   TRAP: "🪤",
   TRASH: "🚮",
+  TREAT: "🍦",
   TREE: "🌳",
-  TREX: "🦖",
+  TRICK: "🎃",
+  TRIM: "💇",
   TROPHY: "🏆",
   TRUCK: "🚙",
-  TRUMPET: "🎺",
   TUBE: "🧪",
   TULIP: "🌷",
+  TURBAN: "👳",
   TURTLE: "🐢",
   TYPE: "⌨️",
   UFOS: "🛸",
-  UNICORN: "🦄",
+  UNICORN: "🦄", // but how?
   UPSET: "😫",
   VEST: "🦺",
   VIDEO: "📼",
   VIOLIN: "🎻",
+  VOLLEY: "🏐",
   VOMIT: "🤮",
   VOTE: "🗳️",
   WAFFLE: "🧇",
+  WALK: "🚶",
   WARN: "⚠️",
   WATCH: "⌚️",
   WATER: "💧",
   WAVE: "🌊",
   WEBS: "🕸️",
   WHALE: "🐳",
+  WHIP: "🥎",
```

Comment:
softball term?

Reply:
... but maybe that's informal. I guess it's fast-pitch, slow-pitch officially. I'll change it to PITCH